Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/SettingsRegistrar.cpp

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "SettingsRegistrar.h"
+
+#include <AzCore/IO/ByteContainerStream.h>
+#include <AzCore/IO/FileIO.h>
+#include <AzCore/Settings/SettingsRegistry.h>
+
+namespace AzToolsFramework
+{
+    AZ::Outcome<void, AZStd::string> SettingsRegistrar::SaveSettingsToFile(
+        const AZStd::string& relativeFilepath,
+        AZ::SettingsRegistryMergeUtils::DumperSettings dumperSettings,
+        const AZStd::string& rootSearchKey) const
```

Comment:
`rootSearchKey` is actually the JSON pointer path root or anchor JSON object where the settings will be saved under.

Reply:
Changed the name to anchorKey to more accurately reflect what this param is for.